GSA/digital-gov-drupal
# Digital-38: migrate shortcodes templates

## Jira ticket




[DIGITAL-](https://cm-jira.usa.gov/browse/DIGITAL-38)

## Purpose



This Pr will set up shortcode templates and update to match Drupal fields.  Also updated PHP files as needed and broke out several short codes into small ones to address special use cases.

## Includes the following PRs that must be merged first

n/a

## Deployment and testing

### Local Setup

`lando rebuild && lando si`

### QA/Testing instructions

1. create a new content type and fill out each shortcode
2. verify that the overall look and function is matching as expect 

## Checklist for the Developer




## Checklist for the Peer Reviewers
